endpoints_management/control/report_request.py

```diff
@@ -237,6 +240,7 @@ def __new__(cls,
                 auth_issuer=u'',
                 auth_audience=u'',
                 backend_time=None,
+                consumer_project_number=SIZE_NOT_SET,
```

Comment:
`SIZE_NOT_SET` doesn't make much sense as a "consumer project number". Can you perhaps rename the constant to `NOT_SET`?

Reply:
Done.